lib/src/timeline.dart

```diff
@@ -552,11 +552,14 @@ class Timeline {
   /// events can already be displayed.
   /// Override the [searchFunc] if you need another search. This will then
   /// ignore [searchTerm].
-  Stream<List<Event>> searchEvent({
+  /// Returns the List of Events and the next prevBatch at the end of the
+  /// search.
+  Stream<(List<Event>, String?)> searchEvent({
```

Comment:
is this worth a breaking change? maybe deprecate this method and create a new one?

Reply:
Good idea. Changed :)